### ISSUE TITLE
Hyprland crashes at launch

### ISSUE THREAD

**Issue by zk4x** [hyprwm/Hyprland#8704]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit 4520b30d498daca8079365bdb909a8dea38e8d55  (version: bump to 0.44.1).
Date: Wed Oct 9 12:54:39 2024
Tag: v0.44.1, commits: 5309
built against aquamarine 0.4.2


flags: (if any)


System Information:
System name: Linux
Release: 6.11.11-300.fc41.x86_64
Version: #1 SMP PREEMPT_DYNAMIC Thu Dec  5 18:38:25 UTC 2024


GPU information: 
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU106 [GeForce RTX 2060 12GB]


os-release: NAME="Fedora Linux"
VERSION="41 (Forty One)"
RELEASE_TYPE=stable
ID=fedora
VERSION_ID=41
```

</details>

### Description

Launched Hyprland from TTY and Hyprland crashed.
This happened after removing python and installing Hyprland.

### How to reproduce

1. install Fedora 41
2. remove python
3. install Hyprland
4. launch Hyprland from TTY

### Crash reports, logs, images, videos

[hyprlandCrashReport0000.txt](https://github.com/user-attachments/files/18122711/hyprlandCrashReport0000.txt)

**Comment by vaxerski** [comment]
mesa bug to me if anything (gallium is mesa iirc)

try disabling explicit sync

**Comment by zk4x** [comment]
> mesa bug to me if anything (gallium is mesa iirc)
> 
> try disabling explicit sync

Disabling explicit sync did not work. I fixed it now, thanks for help. The issue was uninstalling python removed proprietary nvidia driver and Hyprland did not work without it. I tried running sway and cosmic afterwards, cosmic did not work either, but surprisingly sway did work.
Hopefully one day nvidia's drivers will get open sourced properly or AMD will properly support their cards for ML.
Thank you again for your help. zk4x
